### DISCUSSION TITLE
multiple requests

### DISCUSSION THREAD

**Discussion by lauriemann** [qaware/protocurl#188] [Q&A]
is there any way to specify multiple requests in an input file (i.e. to hand pc a bunch of requests to process one after the other)?

**Comment by GollyTicker** [comment]
Hi @lauriemann 

Thanks for your interest. Currently, there is no such thing built into protocurl. When I need this, I simply use it with [xargs](https://www.man7.org/linux/man-pages/man1/xargs.1.html) or simple for loops.

However, perhaps you can show the usecase where you want to use it? Because in that case it may make sense to add this into protocurl.

Also, FYI. I'll be away for a month soon - so please expect delayed responses.

Kinds, GT

**Reply by GollyTicker** [reply]
@lauriemann Ping 😇
